src/type.zig

```diff
@@ -694,7 +694,9 @@ pub const Type = extern union {
             => {}, // The zig type tag is all that is needed to distinguish.
 
             .Pointer => {
+                // Basic hashing for simple types.
                 // TODO implement more pointer type hashing
+                std.hash.autoHash(hasher, self.tag());
```

Comment:
This would unfortunately cause a miscompilation and the real implementation is a bit nuanced. I'll work on this to unblock you.

Reply:
Oh okay, then I guess I’ll just wait for now

Reply:
I have implemented hashing for pointers in a recent master branch commit. If you rebase this PR on master you should be unblocked.

Reply:
Cool, thanks for the heads up :-) I'll open a new one